lib/spack/spack/cmd/load.py

```diff
@@ -62,16 +62,18 @@ def load(parser, args):
              for spec in spack.cmd.parse_specs(args.specs)]
 
     if not args.shell:
-        specs_string = ' '.join(args.specs)
+        specs_str = ' '.join(args.specs) or "SPECS"
         msg = [
-            "This command works best with Spack's shell support",
+            "`spack load` works best with spack's shell support.",
```

Comment:
See the comment associated with the message for environment `activate` and `deactivate`.